_source/logzio_collections/_tracing-sources/dotnet-otel-auto.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Sending traces from .NET applications via auto instrumentation with OpenTelemetry
+logo:
+  logofile: dotnet.svg
+  orientation: vertical
+data-source: Automatic .NET instrumentation
+templates: ["network-device-filebeat"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - new-instrumentation
+order: 1380
+---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Overview](#overview)
+* [Local host](#local-host)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="overview">
+
+Deploy this integration to enable automatic instrumentation of your .NET application using OpenTelemetry.
+
+### Architecture overview
+
+This integration includes:
+
+* Installing the OpenTelemetry .NET instrumentation packages on your application host
+* Installing the OpenTelemetry collector with Logz.io exporter
+* Running your .NET application in conjunction with the OpenTelemetry instrumentation
+
+On deployment, the .NET instrumentation automatically captures spans from your application and forwards them to the collector, which exports the data to your Logz.io account.
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="local-host">
+
+
+### Setup auto-instrumentation for your locally hosted ASP.NET Core application and send traces to Logz.io
+
+**Before you begin, you'll need**:
+
+* An ASP.NET Core application without instrumentation
+* An active account with Logz.io
+* Port `4317` available on your host system
+* A name defined for your tracing service
+
+
+<div class="tasklist">
+
+
+##### Download instrumentation packages
+
+```shell
```

Comment:
This step is not working for me
Im getting an error on it and from the commands docs the syntax should be
dotnet add <PROJECT> package [options] <PACKAGE_NAME>
Or you need to run this command from the app directory?